lib/src/utils/json_converters.dart

```diff
@@ -27,18 +27,15 @@ class StringValueNotifierConverter extends ValueNotifierConverter<String> {
   const StringValueNotifierConverter() : super();
 }
 
-class ListMaterialsValueNotifierConverter
-    implements JsonConverter<ValueNotifier<List<ARKitMaterial>?>, List<Map>?> {
+class ListMaterialsValueNotifierConverter implements JsonConverter<ValueNotifier<List<ARKitMaterial>?>, List<Map>?> {
```

Comment:
Please reformat this and other Dart files with the default "80 characters" line width.